2.6/root/usr/share/container-scripts/mongodb/replset-pre-init.sh

```diff
@@ -0,0 +1,35 @@
+# This file sets some basic mongod replication variables and checks mandatory replication variables
+
+# Add parameters for replication
+export mongod_common_args+=" --oplogSize 64 --replSet ${MONGODB_REPLICA_NAME} "
```

Comment:
You're using `${MONGODB_REPLICA_NAME}` here but validation it's value at the end of the file. Is it ok? Is it possible that it will fail here because of using undeclared variable?